src/entities/FormattingNode/index.ts

```diff
@@ -211,6 +213,51 @@ export class FormattingNode implements InlineNode {
     );
   }
 
+  /**
+   * Removes formatting from the text for a specified inline tool in the specified range
+   *
+   * @param tool - name of inline tool to remove
+   * @param start - char start index of the range
+   * @param end - char end index of the range
+   * @todo Possibly pass data or some InlineTool identifier to relevant only required fragments
+   */
+  public unformat(tool: InlineToolName, start: number, end: number): InlineNode[] {
+    if (this.#tool === tool) {
+      const middleNode = this.split(start);
```

Comment:
why split returns a single node? Maybe it should return an array of nodes? Because after splitting we have 2 nodes

Reply:
That's because split method creates only one new node, you already have a link to the original one:

```
const newNode = node.split(...);

const leftPart = node;
const rightPart = newNode;
```

So no need to make output more complex